lib/worker.go

```diff
@@ -402,7 +402,16 @@ func (w *DefaultBaseWorker) runWatchDog(ctx context.Context) error {
 	}
 }
 
-func (w *DefaultBaseWorker) initMessageHandlers(ctx context.Context) error {
+func (w *DefaultBaseWorker) initMessageHandlers(ctx context.Context) (retErr error) {
+	defer func() {
+		if retErr != nil {
+			if err := w.messageHandlerManager.Clean(context.Background()); err != nil {
```

Comment:
Can we use `ctx` here

Reply:
It could be possible that the `ctx` has been canceled.